packages/vchan/vchan.6.0.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Xen Vchan implementation"
+description: """
+This is an implementation of the Xen "libvchan" or "vchan" communication
+protocol in OCaml. It allows fast inter-domain communication using shared
+memory.
+"""
+maintainer: "jonathan.ludlam@eu.citrix.com"
+authors: ["Vincent Bernardoff" "Jon Ludlam" "David Scott"]
+license: "ISC"
+tags: "org:mirage"
+homepage: "https://github.com/mirage/ocaml-vchan"
+doc: "https://mirage.github.io/ocaml-vchan"
+bug-reports: "https://github.com/mirage/ocaml-vchan/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune"
+  "lwt" {>= "2.5.0"}
+  "cstruct" {>= "1.9.0"}
```

Comment:
```
```suggestion
  "cstruct" {>= "6.0.0"}
```
For
```
#=== ERROR while compiling vchan.6.0.1 ========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/mirage/ocaml-vchan/releases/download/v6.0.1/vchan-6.0.1.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/vchan.6.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p vchan -j 31
# exit-code            1
# env-file             ~/.opam/log/vchan-8363-99c8eb.env
# output-file          ~/.opam/log/vchan-8363-99c8eb.out
### output ###
#       ocamlc lib/.vchan.objs/vchan__Endpoint.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.vchan.objs -I lib/.vchan.objs/.private -I /home/opam/.opam/4.08/lib/base/caml -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cstruct -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/io-page -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/mirage-flow -I /home/opam/.opam/4.08/lib/parsexp -I /home/opam/.opam/4.08/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/xenstore -I /home/opam/.opam/4.08/lib/xenstore/client -intf-suffix .ml -no-alias-deps -open Vchan -o lib/.vchan.objs/vchan__Endpoint.cmo -c -impl lib/endpoint.pp.ml)
# File "lib/endpoint.ml", line 282, characters 12-26:
# 282 |   let len = Cstruct.length buf in
#                   ^^^^^^^^^^^^^^
# Error: Unbound value Cstruct.length
#     ocamlopt lib/.vchan.objs/vchan__Endpoint.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -g -I lib/.vchan.objs -I lib/.vchan.objs/.private -I /home/opam/.opam/4.08/lib/base/caml -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cstruct -I /home/opam/.opam/4.08/lib/fmt -I /home/opam/.opam/4.08/lib/io-page -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/mirage-flow -I /home/opam/.opam/4.08/lib/parsexp -I /home/opam/.opam/4.08/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/sexplib -I /home/opam/.opam/4.08/lib/sexplib0 -I /home/opam/.opam/4.08/lib/xenstore -I /home/opam/.opam/4.08/lib/xenstore/client -intf-suffix .ml -no-alias-deps -open Vchan -o lib/.vchan.objs/vchan__Endpoint.cmx -c -impl lib/endpoint.pp.ml)
# File "lib/endpoint.ml", line 282, characters 12-26:
# 282 |   let len = Cstruct.length buf in
#                   ^^^^^^^^^^^^^^
# Error: Unbound value Cstruct.length
```